lib/generators/active_admin/webpacker/templates/active_admin.js

```diff
@@ -0,0 +1,19 @@
+// import 'active_admin/assets/stylesheets/active_admin'
```

Comment:
These comments in this file seem to be misleading. Should we remove them all and just keep the actual code?

Reply:
Good point, comments cleaned/updated 👍